.github/workflows/release.yaml

```diff
@@ -124,6 +128,46 @@ jobs:
     - name: Setup Cosign
       uses: sigstore/cosign-installer@v3.7.0
 
+    - name: Download CTF
+      uses: actions/download-artifact@v4
+      with:
+        pattern: 'ctf-aggregated'
+        path: gen/downloaded-ctfs
+    - name: Move CTF into correct directory to be recognized by the release process
+      run: |
+        mkdir -p ${{ github.workspace }}/gen
```

Comment:
doesn't the previously executed download step already create that directory (if it doesn't exist)?